cpp/dolfinx/fem/assemble_matrix_impl.cpp

```diff
@@ -132,19 +134,29 @@ void fem::impl::assemble_cells(
 
   // Iterate over active cells
   PetscErrorCode ierr;
-  const int orientation = 0;
   for (std::int32_t cell_index : active_cells)
   {
     // Get cell coordinates/geometry
     for (int i = 0; i < num_dofs_g; ++i)
       for (int j = 0; j < gdim; ++j)
         coordinate_dofs(i, j) = x_g(cell_g[pos_g[cell_index] + i], j);
 
+    const mesh::MeshEntity cell(mesh, gdim, cell_index);
+
+    Eigen::Array<bool, 1, Eigen::Dynamic> cell_edge_reflections
+        = mesh.topology().get_edge_reflections(cell_index);
```

Comment:
This is performance critical code, we should be careful. I would suggest to set the storage to Eigen::RowMajor in the topology - you really want to extract the contiguous chunk of memory, default eigen's is ColumnMajor...